FtpClient/FtpCli/InitializeSession.cs

```diff
@@ -1,4 +1,6 @@
 using System;
+using System.Diagnostics;
```

Comment:
I couldn't see where the Diagnostics or Threading libraries were being used. If they are, this can be disregarded. If not, then these can be removed.

Reply:
Ezra will be removing the libraries. It doesn't break the program, so approving the merge since everything else looks good.